packages/components/src/components/carousel/carousel-nav.tsx

```diff
@@ -1,8 +1,21 @@
-import classNames from 'classnames';
-import PropTypes from 'prop-types';
 import React from 'react';
+import classNames from 'classnames';
 import Icon from '../icon';
 
+type TNav = {
+    active_index: number;
+    active_bullet_color: string;
+    bullet_color: string;
+    className?: string;
```

Comment:
could you please help to find it? as I see it’s just a className, it can’t be required..

Reply:
Base on my search we just use this component in this file `components/src/components/carousel/carousel.jsx` and we are passing the classname as a prop! so this may be require

Reply:
yes, and there are few other passed props. 
I believe we can use className as an optional, because Nav has its own classes, and additional classname should be passed only if we want to add some new styles. This is common approach for all our components, if I'm not mistaken